hooks/api_server.py

```diff
@@ -35,18 +35,31 @@ def on_get(self, req, resp, scope):
             resp.status = falcon.HTTP_400
 
     def on_post(self, req, resp, scope):
-        payload = req.stream.read().decode("utf-8")
+        raw_data = req.stream.read()
+        payload = raw_data.decode("utf-8")
         self._logger.info("POST scope: %s payload: %r headers: %s", scope, payload, req.headers)
 
         try:
             with self._publisher as publisher:
                 publisher.send_event(scope, payload)
-            resp.status = falcon.HTTP_200
 
+            self._send_to_heroku(raw_data)
```

Comment:
@miso-belica Is it necessary for this to rely on successful Rabbit communication? :-)

Reply:
@harpener Yes, it is. Production leads are priority for us. If Rabbit will be down I don't care about some temporary bot somewhere. I hope we will delete this next week after ReactConf. And you can say Hi to our "bot" m.me/roihunter. If you say beer he will understand you :D